src/DisputeHandler.ts

```diff
@@ -443,195 +289,67 @@ class DisputeHandler {
                     encodedLatestFinalizedState,
                     encodedLatestCorrectState,
                     { gasLimit: 2000000 } //TODO! - gas limit
-                );
-            let txReceipt = await txResponse.wait();
+                )
+                .then((txResponse) => txResponse.wait());
```

Comment:
Why does this have 'await' and '.then'? With txResponse = await challengeDispute() and await txResponse.wait() the single catch block will trigger on error (not that it does anything for now)

Reply:
it still triggers on error since there is no `.catch` on the promise
if you like two `awaits` better then so be it. i thought this style is more suitable here since there was no use for the interim `txRecipt` except for turning it in `txResponse`

Reply:
I'm not sure it works like that, but I'm not home to test at the moment. Either way this is minor and everything else is good so feel free to merge